dependencies.yaml

```diff
@@ -262,6 +262,33 @@ dependencies:
         packages:
           - &cmake_ver cmake>=3.26.4,!=3.30.0
           - ninja
+    specific:
```

Comment:
Copied this compiler block directly from `cugraph`: 

To try to fix this error in the conda devcontainer build:

```text
  CMake Error at /home/coder/.conda/envs/rapids/share/cmake-3.31/Modules/CMakeDetermineCXXCompiler.cmake:48 (message):
    Could not find compiler set in environment variable CXX:
  
    /usr/bin/g++.
  
  Call Stack (most recent call first):
    CMakeLists.txt:39 (project)
```

([build link](https://github.com/rapidsai/cugraph-gnn/actions/runs/12018692566/job/33548472621?pr=79))

Most other RAPIDS repos I checked have something like this, and I found offline conversations where it was suggested that the conda devcontainer builds expect the conda environment to provide compilers.

Reply:
Yes, this is a good fix.